websites/P/Phind/metadata.json

```diff
@@ -0,0 +1,63 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "242621043561201666",
+		"name": "GreenMan36"
+	},
+	"service": "Phind",
+  "description": {
+		"en": "Phind is an AI-powered search engine that provides immediate, detailed answers to various questions. Drawing from multiple internet sources, it ensures up-to-date and accurate information, making the search for answers as simple and interactive as having a conversation with a friend.",
+    "fr": "Phind est un moteur de recherche alimenté par l'IA qui fournit des réponses immédiates et détaillées à diverses questions. En puisant dans plusieurs sources internet, il garantit une information précise et à jour, rendant la recherche de réponses aussi simple et interactive qu'une conversation avec un ami.",
+		"nl": "Phind is een door AI aangedreven zoekmachine die directe, gedetailleerde antwoorden geeft op verschillende vragen. Door informatie te halen uit meerdere internetbronnen, zorgt het voor actuele en nauwkeurige informatie, waardoor het zoeken naar antwoorden zo eenvoudig en interactief is als een gesprek met een vriend.",
+		"de": "Phind ist eine KI-betriebene Suchmaschine, die sofortige, detaillierte Antworten auf verschiedene Fragen liefert. Sie zieht Informationen aus verschiedenen Internetquellen heran und gewährleistet so aktuelle und genaue Informationen, sodass die Suche nach Antworten so einfach und interaktiv ist wie ein Gespräch mit einem Freund.",
+		"es": "Phind es un motor de búsqueda impulsado por IA que proporciona respuestas inmediatas y detalladas a diversas preguntas. Extrayendo información de múltiples fuentes de internet, asegura información precisa y actualizada, haciendo que la búsqueda de respuestas sea tan simple e interactiva como tener una conversación con un amigo.",
+		"ar_SA": "Phind هو محرك بحث مدعوم بالذكاء الصناعي يوفر إجابات فورية ومفصلة على مختلف الأسئلة. باستخدام مصادر متعددة من الإنترنت، يضمن معلومات حديثة ودقيقة، مما يجعل البحث عن الإجابات بسيطًا وتفاعليًا كما لو كنت تجري محادثة مع صديق.",
+		"ru": "Phind - это поисковая система, работающая на основе искусственного интеллекта, которая предоставляет мгновенные подробные ответы на различные вопросы. Используя множество интернет-источников, она обеспечивает актуальную и точную информацию, делая поиск ответов таким простым и интерактивным, как беседа с другом.",
+		"it": "Phind è un motore di ricerca alimentato da IA che fornisce risposte immediate e dettagliate a varie domande. Attingendo da molteplici fonti internet, assicura informazioni aggiornate e accurate, rendendo la ricerca di risposte semplice e interattiva come avere una conversazione con un amico."
```

Comment:
Are these all official? If so where do you get them from, I couldn't find it myself..

Reply:
It's reworded from their about page, as for the translations, Phind made them itself and I'm currently in the process of confirming the translations.
I speak Dutch and English and family and friends speak the others. So far I can confirm the German and Spanish translations are correct although the German translation is a bit literal.

Reply:
I've asked if the shorter about is fine, haven't gotten a word back yet.
I did find someone to confirm the arabic for me, waiting for them to check it.
I also asked some people to check the Russian translation, waiting for someone to check it. 
As for french, it seems correct but the person who checked it isn't a native speaker, so yeah... that.
Haven't found anyone to check the itallian yet but if the arabic, dutch, german, spanish, russian and french are all correct or very close, I'm betting on it that the itallian translation would also be correct enough.

Reply:
Alright I should have multiple people for french, russian and itallian. It will take a bit to get all responses but I'm on it.

Reply:
Here's a list of translations checked so far.
- [ ] French  - Being checked
- [x] Dutch
- [x] German
- [x] Spanish
- [x] Arabic
- [ ] Russian - Being checked
- [ ] Itallian - Possibly being checked, sent a message in a multi-lingual Discord.

Reply:
Dutch is confirmed.

Reply:
> I know, I'm Dutch myself 😅

oops 😂